app/views/admin/chat_channels/index.html.erb

```diff
@@ -29,34 +29,46 @@
   </div>
-  <%= paginate @group_chat_channels %>
-  <table class="table">
-    <thead>
+</div>
+<%= paginate @group_chat_channels %>
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col">Channel Name</th>
+      <th scope="col">Users</th>
+      <th scope="col">Add Users</th>
+      <th scope="col">Remove User</th>
+      <% if current_user.any_admin? %>
+        <th scope="th">Delete Channel</th>
+      <% end %>
+    </tr>
+  </thead>
+  <tbody>
+    <% @group_chat_channels.each do |channel| %>
       <tr>
-        <th scope="col">Channel Name</th>
-        <th scope="col">Users</th>
-        <th scope="col">Add Users</th>
-        <th scope="col">Remove User</th>
-      </tr>
-    </thead>
-    <tbody>
-      <% @group_chat_channels.each do |channel| %>
-        <tr>
-          <td><a href="/connect/<%= channel.slug %>" target="_blank" rel="noopener"><%= channel.channel_name %></td>
-            <td><%= channel.users.pluck(:username).join(", ") %></td>
-            <td>
-              <%= form_for [:admin, channel] do |f| %>
-                <%= f.text_field :usernames_string, id: "usersToAddToChannel#{channel.id}", placeholder: "Usernames to add", aria: { label: "Usernames" } %>
-                <%= f.submit %>
-              <% end %>
-            </td>
+        <td><a href="/connect/<%= channel.slug %>" target="_blank" rel="noopener"><%= channel.channel_name %></td>
+          <td><%= channel.users.pluck(:username).join(", ") %></td>
+          <td>
+            <%= form_for [:admin, channel] do |f| %>
+              <%= f.text_field :usernames_string, placeholder: "Usernames to add" %>
+              <%= f.submit %>
+            <% end %>
+          </td>
+          <td>
+            <%= form_with url: remove_user_admin_chat_channel_url(channel), model: [:admin, channel], method: :delete, local: true do |f| %>
+              <%= f.text_field :username_string, placeholder: "Username to remove" %>
+              <%= f.submit "Remove From channel", data: { confirm: "Are you sure you want to remove this user?" } %>
+            <% end %>
+          </td>
+          <% if current_user.any_admin? %>
             <td>
-              <%= form_with url: remove_user_admin_chat_channel_url(channel), model: [:admin, channel], method: :delete, local: true do |f| %>
-                <%= f.label :usernames_string, "Users to Remove", class: "sr-only" %>
-                <%= f.text_field :username_string, id: "usersToRemoveFromChannel#{channel.id}", placeholder: "Username to remove", aria: { label: "Usernames" } %>
-                <%= f.submit "Remove From channel", data: { confirm: "Are you sure you want to remove this user?" } %>
+              <% if channel.users.count.zero? %>
+                <%= button_to "Delete Channel", { action: "destroy", id: channel.id }, method: :delete, data: { confirm: "Are you sure?" }, class: "crayons-btn crayons-btn--danger" %>
+              <% else %>
+                Cannot delete; channel has users
```

Comment:
**Suggestion (nit-pick, non-blocking)**: this reads a little bit awkward to me with the ; in the middle of the sentence.  Perhaps we can change it to `Channel with users cannot be deleted`